flake.nix

```
@@ -300,24 +304,26 @@
             outputs = [ "out" "dev" "doc" ];
 
             nativeBuildInputs = nativeBuildDeps;
-            buildInputs = buildDeps ++ awsDeps;
+            buildInputs = buildDeps
+              # There were issues building this in the past
```

I bet this comment is from me :sweat_smile: but it is kind of cryptic!

---

```suggestion
              # There have been issues building these dependencies
```
this is a little clearer. I am fine figuring out later if we can cross compile them after all.